docs/using-the-jdbc-driver/using-plugins/UsingTheAwsSecretsManagerPlugin.md

```diff
@@ -12,10 +12,11 @@ The following properties are required for the AWS Secrets Manager Connection Plu
 
 > **Note:** To use this plugin, you will need to set the following AWS Secrets Manager specific parameters.
 
-| Parameter                | Value  |                         Required                         | Description                                             | Example     | Default Value |
-|--------------------------|:------:|:--------------------------------------------------------:|:--------------------------------------------------------|:------------|---------------|
-| `secretsManagerSecretId` | String |                           Yes                            | Set this value to be the secret name or the secret ARN. | `secretId`  | `null`        |
-| `secretsManagerRegion`   | String | Yes unless the `secretsManagerSecretId` is a Secret ARN. | Set this value to be the region your secret is in.      | `us-east-2` | `us-east-1`   |
+| Parameter                | Value  |                         Required                         | Description                                                              | Example                 | Default Value |
+|--------------------------|:------:|:--------------------------------------------------------:|:-------------------------------------------------------------------------|:------------------------|---------------|
+| `secretsManagerSecretId` | String |                           Yes                            | Set this value to be the secret name or the secret ARN.                  | `secretId`              | `null`        |
+| `secretsManagerRegion`   | String | Yes unless the `secretsManagerSecretId` is a Secret ARN. | Set this value to be the region your secret is in.                       | `us-east-2`             | `us-east-1`   |
+| `secretsManagerEndpoint` | String |                            No                            | Set this value to be the endpoint override to retrieve your secret from. | `http://localhost:1234` | `null`        |
```

Comment:
We need to indicate that it is a URL in the description, and also indicate which formats are OK or not (ex: port mandatory or not)